### PR TITLE
GitHub sync

### LINKED ISSUE #301: Error: Argument type (Iterable) can’t be assigned to parameter (List<dynamic>) type
I think this might be a new bug in the SDK or in `angular_components`, if it isn’t a bug in [my code](https://github.com/4cm4k1/website), but want to check here before filing an issue on the SDK…

**Environment:**
* macOS 10.14
* Dart SDK 2.0.0-dev.69.3
* `angular_components` 0.9.0-beta+1

**Output of `webdev build`:**
```shell
[SEVERE] build_web_compilers|entrypoint on web/main.dart:
Dart2Js finished with:

packages/angular_components/src/utils/angular/scroll_host/scroll_host_base.dart:83:27:
Error: The argument type '(dart.core::Iterable<dart.dom.html::IntersectionObserverEntry>, dart.dom.html::IntersectionObserver) → void' can't be assigned to the parameter type '(dart.core::List<dynamic>, dart.dom.html::IntersectionObserver) → void'.

Error: Compilation failed.
```

Please let me know if you need additional info. Thanks! 😄

### PR DESCRIPTION
**Pull request by nshahan**
Updated gallery branch https://github.com/dart-lang/angular_components_example/tree/github-sync

Fixes #301

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,35 @@
...
+* Remove `SelectableChangeNotifier` and deprecate `SelectableWithComposition`
+  with the intention to remove. They are widely unused, complicated the
+  implementations, and unsound in Dart 2.
+
+#### Material Dialog
+* Fix bug where a disposable could be added to its disposer after it had already
+  been destroyed.
+
+#### Material Icon
+* Add aria label and use them for trailing/leading icons of Material Input.
```
Comment (chalin) [inline CHANGELOG.md]: At some point we should make a global search and replace in this file (and the API docs) of: /\baria\b/ -> ARIA

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,35 @@
...
+### Other Updates
+#### Miscellaneous
+* Add `is{Selectable|Disabled|Hidden}In()`, `getOptionIn()` and, `filterWhere()`
+  as static helpers to `Selectable`.
+* Add a lookup method for finding the closest Material Color name based on any
+  input color.
+
+#### Dart 2 Updates
+* Fixes for Dart 2 compile-time and runtime errors.
+* Applicaion of `dartfmt --fix`.
```
Comment (chalin) [inline CHANGELOG.md]: Applicaion of -> Run

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,35 @@
...
+
+#### Material Dialog
+* Fix bug where a disposable could be added to its disposer after it had already
+  been destroyed.
+
+#### Material Icon
+* Add aria label and use them for trailing/leading icons of Material Input.
+
+#### Material Popup
+* Change `<main>` html tag to a `<div>`. HTML5 states there should only be one
```
Comment (chalin) [inline CHANGELOG.md]: html tag -> HTML tag

File: pubspec.yaml [hunk 4]
```diff
@@ -8,8 +8,8 @@ author: Dart Team <misc@dartlang.org>
 environment:
```
Comment (chalin) [inline pubspec.yaml]: From another thread, @nshahan wrote:

> Homepage -> https://webdev.dartlang.org/components
> Docs -> https://webdev.dartlang.org/api?package=angular_components

Will you be adding the `documentation` entry in this release?

**Comment by chalin** [comment]
Can you also address #305 in the context of this PR? /cc @kwalrath

**Comment by TedSander** [comment]
@nshahan is out today. I'm a little concerned about not releasing this till Monday as people are broken at head. I'm thinking about just merging and pushing. Thoughts?

**Comment by chalin** [comment]
> merging and pushing

SGTM

**Pull request merged**